src/RequestorPages/NewJobPage.js

```diff
@@ -51,8 +51,9 @@ export default class NewJobPage extends Component {
             title: this.state.title,
             jobType: this.state.jobType,
             description: this.state.description,
-            startDateTime: this.state.startDateTime,
```

Comment:
These attributes for a job should still be named "startDateTime" and "endDateTime"!